terraform/region/modules/region/waf.tf

```diff
@@ -84,6 +123,18 @@ resource "aws_wafv2_web_acl" "main" {
   }
 }
 
+// Avoid false-positives for the AWSManagedRulesCommonRuleSet rule - see AWS support ticket 11374935181
+resource "aws_wafv2_regex_pattern_set" "allow_on_keyword" {
+  name        = "on-keyword"
+  description = "On keyword regex pattern set"
+  scope       = "REGIONAL"
+
+  regular_expression {
+    regex_string = ".*[oO][nN].*=.*$"
```

Comment:
This will match for multipart form encoded POSTs, which is fine as we're no longer bothering with converting the popup Ajax requests to JSON.